ansible/roles/dashd/tasks/main.yml

```diff
@@ -56,6 +56,26 @@
     - docker-compose.yml
     - .env
 
+- name: Configure log rotation
+  include_role:
+    name: arillso.logrotate
+  vars:
+    logrotate_use_hourly_rotation: true
+    logrotate_applications:
+      - name: core-logs
+        definitions:
+          - logs:
+              - "{{ dashd_home }}/.dashcore/{{ 'devnet-' + dash_devnet_name if dash_network == 'devnet' else 'testnet3' }}/debug.log"
```

Comment:
Is there some reason to construct the network name instead of using the `dash_network_name` variable?

Reply:
Good point!